src/trace-writer.js

```diff
@@ -145,6 +151,16 @@ TraceWriter.prototype.getProjectId = function(callback) {
     property: 'project-id',
     headers: headers
   }, function(err, response, projectId) {
+    if (response && response.statusCode !== 200) {
+      if (response.statusCode === 503) {
+        err = new Error('Metadata service responded with a 503 status ' +
```

Comment:
I know this error message was copied but I don't think the `please try again` is relevant for users. They have not taken any manual action that they could try to repeat.